### ISSUE TITLE
Accessibility Issue: Cannot edit form fields using mobile screen readers

### ISSUE THREAD

**Issue by thecoleorton** [square/connect-api-examples#100]
We noticed an accessibility issue when editing the Square Payment Form fields with a screen reader turned on. The first time through the form, it's not hard to select the four CC fields and enter data. However, if while editing, it can be impossible to select some of the fields with VoiceOver/TalkBack on. The really challenging part is that it sounds like you've selected it, but if you look at it you see that focus hasn't been moved to the field that was stated with VoiceOver.

**Comment by StephenJosey** [comment]
Thanks for letting us know @thecoleorton! We will look into this and will let you know once we have an update or more information.

**Comment by thecoleorton** [comment]
👍
